proposals/compatibility-mode.md

```diff
@@ -103,12 +119,16 @@ Calls to `createTexture()` or `createBindGroupLayout()` with this combination ca
 
 **Justification**: GLSL ES 3.1 (section 4.4.7, "Format Layout Qualifiers") does not permit any two-component (RG) texture formats in a format layout qualifier.
 
+Feature: `storage-texture-rg`.
```

Comment:
Possible that we would need to be able to expose these separately, some hardware might have `rg32float` but not `rg32uint`/`rg32sint` or vice versa. Not sure how likely.

Reply:
Noted.

If we did that, though, it'd be tempting to split up `depth-texture-load` and `depth-texture-sample` into the explicit depth formats. But that's a lot of features.